### PR TITLE
Category pills for search + plugin pages

### PR DESCRIPTION
**Pull request by klai95**
`Ticket`
https://app.shortcut.com/imaging/story/172045/category-pills-for-search-plugin-pages
`Screenshots`
Screen Shot 2021-12-13 at 3 12 55 PM
Screen Shot 2021-12-13 at 3 13 30 PM

### CODE REVIEW

File: frontend/src/components/PluginSearch/PluginSearchResult.tsx [hunk 1]
```diff
@@ -193,6 +195,18 @@ export function PluginSearchResult({
             />
           </ul>
 
+          {/* Plugin categories */}
+          <ul className="mt-5 text-xs">
+            <SkeletonLoader
+              render={() =>
+                isObject(plugin.category) &&
+                Object.entries(plugin.category).map(([pluginDimension]) => (
+                  <CategoryChip dimension={pluginDimension as HubDimension} />
```
Comment (codemonkey800) [inline frontend/src/components/PluginSearch/PluginSearchResult.tsx:204]: I noticed you're not passing `category` anywhere here, so the chips won't actually render 😢

Comment (klai95) [reply frontend/src/components/PluginSearch/PluginSearchResult.tsx:204]: oh yeah I wasn't done haha I just wanted to fix the test errors first as I figure out how to map categories to category

File: frontend/src/components/PluginSearch/PluginSearchResult.tsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 import { Link } from '@/components/common/Link';
 import { SkeletonLoader } from '@/components/common/SkeletonLoader';
 import { TextHighlighter } from '@/components/common/TextHighlighter';
 import { useLoadingState } from '@/context/loading';
 import { SearchResultMatch } from '@/store/search/search.types';
-import { PluginIndexData } from '@/types';
+import { HubDimension, PluginIndexData } from '@/types';
 import { formatDate, formatOperatingSystem } from '@/utils';
 
+import { CategoryChip } from '../CategoryChip';
```
[the remark this answers is not in the document]
Comment (klai95) [reply frontend/src/components/PluginSearch/PluginSearchResult.tsx:13]: Will do! I will be following that format moving forward

**Comment by codemonkey800** [comment]
@liaprins-czi in case you're wondering why I added you and removed you 😆 I was hoping for you to review the design for this, but let's save that for a future PR when I add the tooltip copy

**Review by codemonkey800 — APPROVED** [review]
LGTM! this is awesome, great work 😄

**Pull request merged**